### DISCUSSION TITLE
API versioning ?

### DISCUSSION THREAD

**Discussion by dkamali-mc** [directus/directus#7220] [Archive]
Hey everyone, 

Wondering if API versioning is possible with directus? if not is it on the road map? 

Thanks

**Comment by rijkvanzanten** [comment]
What sort of API versioning?

**Reply by dkamali-mc** [reply]
I meant scoping, `v1`, `v2` for generated endpoints.

**Reply by rijkvanzanten** [reply]
Generated endpoints? You can name custom endpoints whatever you like, so you can add a `v1` identifier as part of the path 🙂
